### PR TITLE
[logshipper] Debug helm chart test

### PR DESCRIPTION
**Pull request by ibakshay**

### CODE REVIEW

**Comment by ibakshay** [comment]
Hi @IvoGoman, 

Could you please help with this?
 The `ct install --config .github/configs/helm-test.yaml --target-branch main` command in the `helm-lint-test` workflow is failing. The OpenSearch domain  [referenced here](https://github.com/cloudoperators/greenhouse-extensions/blob/debug-logshipper-test/logshipper/chart/ci/test-values.yaml#L6-L11)  is incorrect, preventing the `log-shipper` pod from reaching it.

```
[2024/09/27 16:50:57] [ warn] [net] getaddrinfo(host='my.log.sink', err=4): Domain name not found
[2024/09/27 16:50:57] [ warn] [engine] failed to flush chunk '1-1727455856.277850384.flb', retry in 6 seconds: task_id=0, input=tail.0 > output=opensearch.0 (out_id=0)
[2024/09/27 16:50:58] [ warn] [net] getaddrinfo(host='my.log.sink', err=4): Domain name not found
```

This is causing the test to fail.

**Comment by IvoGoman** [comment]
We do not have an OpenSearch instance which is available from Github :/ The values are foo to be able to check that the chart renders correctly.

File: logshipper/chart/templates/fluent-bit-configmap.yaml [hunk 1]
```diff
@@ -44,51 +43,6 @@ data:
...
-      [FILTER]
-          Name               kubernetes
-          Match              default_kube.*
-          Kube_Tag_Prefix    kube.var.log.containers.
-          Kube_URL           https://kubernetes.default.svc:443
-          Kube_CA_File       /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-          Kube_Token_File    /var/run/secrets/kubernetes.io/serviceaccount/token
-          Merge_Log          On
-          Merge_Log_Trim     On
-
```
Comment (IvoGoman) [inline logshipper/chart/templates/fluent-bit-configmap.yaml:91]: This is a breaking change. What is the reason for removing this?

File: logshipper/chart/templates/fluent-bit-configmap.yaml [hunk 2]
```diff
@@ -120,17 +74,6 @@ data:
...
-      @INCLUDE custom-outputs.conf
-{{- end }}
-
-      [OUTPUT]
-          Name            prometheus_exporter
-          Match           internal_metrics
-          Host            0.0.0.0
-          Port            2021
-          Add_label       app {{ include "fluent-bit.name" . }}
-
```
Comment (IvoGoman) [inline logshipper/chart/templates/fluent-bit-configmap.yaml:133]: This is a breaking change. What is the reason for removing this?

**Comment by ibakshay** [comment]
We shall close this PR for now. I was playing around with log-shipper to fix a helm chart test related issue.

**Pull request closed without merging**